install_config/install/advanced_install.adoc

```diff
@@ -2216,22 +2206,35 @@ ifdef::openshift-origin[]
 endif::[]
 ----
 <1> Sets the name for the systemd service.
-<2> Specify the location for your inventory file on your local workstation.
+<2> Specify the location for your local inventory file.
```

Comment:
This is the only place we referred to local workstation... this can run anywhere that docker and systemd run, not that likely to be a workstation actually.